src/js/slideshow.js

```diff
@@ -176,77 +169,66 @@
 					void c.el.offsetWidth; // Trigger reflow to restart animation
 					c.el.classList.add( 'reboundLeft' );
 					return;
-				} else {
-					index++;
 				}
+				index++;
 			}
 		}
 
 		setPosition( c, index, 'smooth' );
-
 	}
 
-	/*
+	/**
 	 * Set position of slideshow
-	 * @param c obj the carousel object
-	 * @param index int the index to move to
+	 *
+	 * @param {Object} c The carousel object.
+	 * @param {number} index The index to move to.
+	 * @param {string} behavior The scroll behavior.
 	 */
 	function setPosition( c, index, behavior ) {
-
 		c.el.scroll(
 			{
 				top: 0,
 				left: c.el.offsetWidth * index,
-				behavior: behavior
-		}
-			);
+				behavior,
+			}
+		);
 
 		c.el.setAttribute( 'data-position', index );
 
 		updateCounter( c, index );
 		updateActive( c, index );
-
 	}
 
 	function updateCounter( c, index ) {
-		c.counter.innerHTML = '<span>' + ( index * 1 + 1 ) + '</span> of ' + c.n;
+		c.counter.innerHTML = '<span>' + ( ( index * 1 ) + 1 ) + '</span> of ' + c.n;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unsafe assignment to innerHTML](https://app.codacy.com/manual/uriweb/uri-component-library/pullRequest?prid=4811013)